app/src/main/java/org/kinecosystem/kinit/viewmodel/info/InfoViewModel.kt

```diff
@@ -40,7 +42,12 @@ class InfoViewModel(val navigator: Navigator) : TabViewModel {
     }
 
     fun onStartBackupClicked(view: View) {
-        navigator.navigateTo(Navigator.Destination.WALLET_BACKUP)
+        if(isBackedUp.get()){
+            showCreateNewBackupAlert.set(true)
+            showCreateNewBackupAlert.set(false)
```

Comment:
that doesn't look right. kind of hacking around the usage of binding properties... 
I would prefer to add the ability of displaying the popup to the navigator

Reply:
How about something like:
navigator.navigateUponConfirm(Navigator.Destination.WALLET_BACKUP, AlertToDisplay)

Reply:
not want to use the context in the model and nagivator is not alert dispalyer
but i solved it in another way